### PR TITLE
`loaders.load_csv`: allow relative paths w/ `ODOO_DATA_PATH`

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: anthem/lyrics/loaders.py [hunk 1]
```diff
@@ -3,16 +3,24 @@
...
 
 def load_csv(ctx, model, path, header=None, header_exclude=None, **fmtparams):
-    """ Load a CSV from a filename
+    """Load a CSV from a file path.
+
+    :param ctx: Anthem context
+    :param model: Odoo model name or model klass from env
+    :param path: absolute or relative path to CSV file.
+        If a relative path is given you must provide a value for
+        `ODOO_DATA_PATH` in your environment
```
Comment (guewen) [inline anthem/lyrics/loaders.py]: "or set the `--odoo-data-path` option"

File: anthem/cli.py [hunk 2]
```diff
@@ -53,9 +57,21 @@ def main():
         'configuration file and -d for the database name. Those must be at '
         'the end of the arguments'
     )
+    parser.add_argument(
+        'odoo-data-path',
+        nargs=argparse.REMAINDER,
```
Comment (guewen) [inline anthem/cli.py]: `REMAINDER` is to take all the "unknown" remaining args, which will be passed down to odoo, only `odoo-args` should use it. This argument should probably only have `action='store_true'` and no `nargs`

**Comment by guewen** [comment]
Travis says:

```
usage: anthem [-h] [-i] [-q] target ... odoo-data-path
anthem: error: too few arguments
```

**Comment by simahawk** [comment]
@guewen damn, somehow I lost the `--` before arg name :( Should be ok now

**Pull request merged**